containers/signing-service/install-dependencies.sh.j2

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
+
+# Update package lists for Debian and install dependencies
+dependencies=(
+
+    # Packages needed to build python extensions
+    gcc
+    git
+    software-properties-common
+)
+
+testing_dependencies=(
+    # There's no stable Python 3.6 package in Debian Stretch yet so we have to get it from testing.
+    python{{ python_version }}
+    python{{ python_version }}-dev
+    virtualenv
+)
+
+apt-get update --assume-yes
+apt-get install              \
+    --assume-yes             \
+    --no-install-recommends  \
+    ${dependencies[*]}
+
+add-apt-repository "deb {{ debian_mirror }} testing main"
+apt-get update --assume-yes
+
+apt-get install              \
+    --target-release testing \
+    --assume-yes             \
+    --no-install-recommends  \
+    ${testing_dependencies[*]}
+
+# Add non-privileges user
+groupadd --gid 999 signingserviceuser
+useradd --system --uid 999 --gid signingserviceuser signingserviceuser
```

Comment:
Looks good. Let's use it more.

But I'd put it in a separate script called `create-user.sh` and run it before everything else.

Reply:
Done.